lighthouse-core/config/default-config.js

```diff
@@ -459,7 +458,6 @@ const defaultConfig = {
         {id: 'aria-toggle-field-name', weight: 3, group: 'a11y-aria'},
         {id: 'aria-valid-attr-value', weight: 10, group: 'a11y-aria'},
         {id: 'aria-valid-attr', weight: 10, group: 'a11y-aria'},
-        {id: 'audio-caption', weight: 10, group: 'a11y-audio-video'},
```

Comment:
I think we're still running this audit because of its `wcag2a` tag though, so I think we'll want to explicitly say `enabled: false` in our accessibility gatherer.

https://github.com/GoogleChrome/lighthouse/blob/40a187f1cc8420c70ecfc6e37b192f368eceafc3/lighthouse-core/gather/gatherers/accessibility.js#L34-L51